src/sam-utils.js

```diff
@@ -1,48 +1,35 @@
 const { samRoot } = require('./config')
 const { fetchOk, Response } = require('./utils')
-const { jwtDecode, InvalidTokenError }   = require('jwt-decode')
 const NodeCache = require('node-cache')
 
 const samUserCache = new NodeCache({ stdTTL: 300, checkperiod: 300, maxKeys: 10000 })
 
-async function getSamUser(req, userEmail) {
+async function getSamUser(req) {
   const res = await fetchOk(
     `${samRoot}/register/user/v2/self/info`,
     { headers: { authorization: req.headers.authorization }, serviceName: 'auth' }
   )
   const samUser = await res.json()
-  samUserCache.set(userEmail, samUser)
-  req.user = samUser
-  if (!req.user.enabled) {
-    throw new Response(403, 'Forbidden')
-  }
```

Comment:
i moved this up to the calling fn just because i figured it made a tad more sense